spotinst-cli

```diff
@@ -464,8 +463,12 @@ for group, group_values in groups_to_update.iteritems():
     print message
     res = query_api(req_path, token, req_type, payload)
     if res.status_code == 200:
-        if req_type == 'GET':
+        if req_type == 'GET' and options.show_response == 'True':
+            print res.content
+    if res.status_code == 200:
+        if req_type == 'POST' and options.show_response == 'True':
             print res.content
```

Comment:
why do we need to print "res" and "res.content"?

Reply:
we actually dont need res ill remove it
res.content shows the content of the response